services-js/commissions-search/src/App.js

```diff
@@ -50,13 +52,40 @@ class App extends React.Component {
     this.setState({ currentSeats: event.target.value });
   };
 
+  findCheckedBoxes4GA = event => {
+    var chk_arr = document.getElementsByClassName('cb');
+    var chklength = chk_arr.length;
+    for (var k = 0; k < chklength; k++) {
+      var checkit = chk_arr[k].firstChild.checked ? true : false;
```

Comment:
This data should already be present in the component’s state, so you should get it from there rather than inspecting HTML elements.

Also, avoid `document`-based selection in general in React, since you could be grabbing input elements from other places on the page. If you reeeeeaaaaalllly need native DOM operations, use a Ref to get access to an HTML element specifically owned by your component.

Reply:
Per our meeting / conversation, this was / is not available in the component state and would have to be altered using a rather buried Apollo set up.

Reply:
Can you put that in the code as a comment?

It's much more likely to be found there when someone needs to look back on the code than digging up the old PR.

Reply:
I ended up adding an attribute to the checkbox input elements in the FacetList.js file that displays the label value. Having these values available directly in the DOM we now can easily access them and add to REACT state in the App.js file. From here we can iterate over the new state Object (checkedNames) upon submission of the filter and send to GA.